BUILD.md

```diff
@@ -102,6 +102,20 @@ Follow the instructions below to execute a Docker-based build and execution.
 5. To execute test cases refer to [Testing](#testing) section below
 6. To exit the Avalon program, press `Ctrl-c`
 
+**Running multiple worker pools together**
+
+To run multiple worker pools together, modify `docker-compose-pool.yaml`. It also has a corresponding docker compose file `docker-compose-pool-sgx.yaml` for running in Intel SGX hardware mode. This setup starts 1 pool of workers by default with the composition:
```

Comment:
This setup starts 1 pool of workers -> This setup starts a worker pool with the following configuration: